Sources/Stencil/AssetsCatalogContext.swift

```diff
@@ -38,11 +38,19 @@ extension AssetsCatalogParser {
       switch entry {
       case let .group(name: name, items: items):
         return [
+          "type": "group",
           "name": name,
           "items": structure(entries: items)
         ]
+      case let .color(name: name, value: value):
+        return [
+          "type": "color",
+          "name": name,
+          "value": value
```

Comment:
yeah, so might be better to leave it as is? It's not like `value` is a bad name, it's the value you eventually pass to `UIImage(named: )` or `UIColor(named: )`, the rest is window dressing.